src/Views/AccountView.vala

```diff
@@ -62,10 +64,14 @@ public class Installer.AccountView : AbstractInstallerView {
             input_purpose = NAME
         };
 
+        ((Gtk.Label) realname_label.get_first_child ()).mnemonic_widget = realname_entry;
```

Comment:
I believe that it should be something like:
```vala
realname_entry.update_relation (Gtk.AccessibleRelation.LABELLED_BY, realname_label);
```
I haven't had the time to test it though

Reply:
So the docs say:
> setting the mnemonic widget on a GtkLabel will update the GTK_ACCESSIBLE_RELATION_LABELLED_BY relation on the widget with a reference to the label

Which I read as being a recommendation

Reply:
It is just that I don't like the use of `realname_label.get_first_child ()` we should do something in granite for this

Reply:
Yeah I filed https://github.com/elementary/granite/issues/662. I'll make a branch today